docs/source/quickstarts/gke.md

```diff
@@ -0,0 +1,91 @@
+# Deploying ScyllaDB on GKE
+
+This is a quickstart guide to help you set up a basic GKE cluster quickly with local NVMes and solid performance.
+
+This is by no means a complete guide, and you should always consult your provider's documentation.
+
+
+## Creating a GKE cluster
+
+First, we need to create a kubelet config to configure [static CPU policy](../installation/kubernetes/generic.md#static-cpu-policy)
+:::{code} bash
+:number-lines:
+
+cat > systemconfig.yaml <<EOF
+kubeletConfig:
+  cpuManagerPolicy: static
+EOF
+:::
+
+
+Then we'll create a GKE cluster with the following:
+
+:::{code} bash
+
+gcloud container \
+clusters create 'my-k8s-cluster' \
+--zone='us-central1' \
+--machine-type "n1-standard-8" \
+--num-nodes "2" \
+--disk-type "pd-ssd" --disk-size "20" \
+--image-type "UBUNTU_CONTAINERD" \
+--enable-stackdriver-kubernetes
+:::
+
+and then we'll a dedicated pool with NVMes for ScyllaDB
+
+:::{code} bash
+
+gcloud container \
+node-pools create "scyllacluster-pool" \
+--cluster 'my-k8s-cluster' \
+--machine-type "n1-standard-32" \
+--num-nodes "4" \
+--disk-type "pd-ssd" --disk-size "20" \
+--local-nvme-ssd-block count="8" \
+--image-type "UBUNTU_CONTAINERD" \
+--system-config-from-file=systemconfig.yaml \
+--no-enable-autoupgrade \
+--no-enable-autorepair \
+--node-labels scylla.scylladb.com/node-type=scylla \
+--node-taints scylla-operator.scylladb.com/dedicated=scyllaclusters:NoSchedule
```

Comment:
the clusters in the generic section don't have the corresponding toleration, it should be mentioned on this page imo

Reply:
I've extended the default example to include them and added a section there about placement